web-client/tailwind.config.js

```diff
@@ -23,32 +22,6 @@ module.exports = {
       },
     },
   },
-  variants: {
```

Comment:
@PiDelport, now that tailwind css v3.0 automatically enables all variants. how would we go about disabling variants that are not needed for the wallet (unless the wallet makes use of all the variants)?

Reply:
I think with Tailwind v3.0, the JIT engine by default takes care of only generating the variants that are actually used, so we don't need to do anything special for this:

- <https://tailwindcss.com/blog/tailwindcss-v3#just-in-time-all-the-time>